strapi-react-app/react-frontend/package.json

```diff
@@ -30,5 +30,23 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "husky": "^4.2.5",
+    "lint-staged": "^10.2.6",
+    "typescript": "^3.9.3"
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx}": [
+      "eslint . --fix"
+    ],
+    "src/**/*.scss": [
+      "stylelint --syntaxscss --fix"
```

Comment:
looks like you are referencing a package that doesn't exist here `stylelint`

Reply:
Will remove the .css checker for now and integrate when we actually have files